Search/Adapter/ElasticSearchAdapter.php

```diff
@@ -78,6 +78,11 @@ public function index(Document $document, $indexName)
                 case Field::TYPE_ARRAY:
                     $fields[$massiveField->getName()] = $value;
                     break;
+                case Field::TYPE_DATE:
```

Comment:
https://github.com/massiveart/MassiveSearchBundle/blob/develop/Resources/docs/extending.rst#converters

@danrot could it be that the docs wont be updated automatically?